src/DataFactory/RecordDataFactory.js

```diff
@@ -36,10 +36,7 @@ class RecordDataFactory extends BaseDataFactory {
             return resolvedValue[0]
         }
 
-        return new FateTuple(
-            FateTypeRecord(type.keys, type.valueTypes),
-            resolvedValue
-        )
+        return new FateTuple(FateTypeRecord(type.keys, type.valueTypes), resolvedValue)
```

Comment:
Single line shouldn't be enforced in this case as well

Reply:
There is no good way to implement this: we can reduce `printWidth` or add a comment as the above 🤷‍♀️